spotify_tensorflow/tf_record_spec_parser.py

```diff
@@ -44,16 +59,16 @@ def parse_tf_record_spec(cls, tf_record_desc_path, dir_path):
         assert len(feature_info) > 0, "TFRecordSpec parsing error: No feature found."
 
         # groups by multispec
-        multispec_feature_groups = []
+        multispec_feature_groups = []  # type: List[List[str]]
         if "multispec-id" in feature_info[0][2]:
-            d = OrderedDict()
+            d = OrderedDict()  # type: OrderedDict[int, List[str]]
             for name, _, tags in feature_info:
                 key = int(tags["multispec-id"])
                 if key not in d:
                     d[key] = []
                 d[key].append(name)
-            multispec_feature_groups = [None] * len(d)
-            for i, f in d.items():
+            multispec_feature_groups = [[str()]] * len(d)
```

Comment:
Is this change fine @richwhitjr @yonromai ?

Reply:
These values are overridden by the loop so I guess it's functionally identical

Reply:
Why did you do this change BTW?

Reply:
@yonromai typing requires it - otherwise it fails mypy cause list of lists of nones is not a list of lists of strings.

Reply:
👍

Reply:
Looks good for me also.  Could be slightly more expensive this way but shouldn't be a huge deal.